File: food_ordering_app/app/src/main/java/com/example/food_ordering_app/MainActivity.java
package com.example.food_ordering_app;

import androidx.annotation.RequiresApi;
import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Build;
import android.os.Bundle;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {
    private RecyclerView recyclerView;



    private LinearLayoutManager manager;

    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        utils.changeStatusBarAndNavigationBarColor(MainActivity.this,R.color.mirage,R.color.mirage_dark);



    }

}

File: BirthdayApp/app/src/main/java/com/example/birthdayapp/MainActivity.kt
package com.example.birthdayapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }

    fun createWish(view: View) {


        val name = name.editableText.toString()
        //Toast.makeText(this,"happy anniversary $name ",Toast.LENGTH_LONG).show()
        val intent=Intent(this,Greetingpage::class.java)
       intent.putExtra(Greetingpage.Name_Extra,name)
        startActivity(intent)

    }
}